Odata-docs/webapi-8/fundamentals/query-options.md

```diff
@@ -0,0 +1,486 @@
+---
+title: "OData query Options"
+description: "This documentation describes how to use the supported query options."
+
+author: KenitoInc
+ms.author: kemunga
+
+ms.date: 12/16/2022
+---
+# OData query options
```

Comment:
Your doc is being flagged for having a H1 that is conflicting with other docs. You could fix that by renaming this to **OData Query Options in ASP.NET Core OData 8** or **Working with OData Query Options in ASP.NET Core OData 8**. If you make the change, be sure to change the **title** field at the top too